models/index.js

```diff
@@ -28,6 +28,67 @@ exports.selectArticles = () => {
     });
 };
 
+exports.sortArticles = (sorter=`created_at`, order=`DESC`, topic) => {
+    const sortParams = ['author',
+                        'title',
+                        'article_id',
+                        'topic',
+                        'created_at',
+                        'votes',
+                        'comment_count'];
+    const orderParams = ['ASC', 'DESC'];
+    const topicParams = ['mitch', 'cats'];
```

Comment:
At the moment, this will work for your test data. 

What happens when you use the dev data and these topics dont exist? 
 You need to dynamically check for valid topics from the current database. 
 
 The topics will always be changing/being added to and this should scale to account for that. 
 
 At the moment, this wont work on the front end, with the development data. You wont be able to filter by topic.